docs/scripts.rst

```diff
@@ -130,6 +130,10 @@ Explanation for the keys:
 - ``urls``: dictionary of table names and the respective urls
 - ``version``: "1.0.0"
 
+``fields``: Although retriever automatically detects the schema of the table,
```

Comment:
`pk-auto: Auto generated primary key starting from 1`
This includes addition of an extra column of type int. right? what is it's name?